charts/alertmanager/templates/services.yaml

```
@@ -4,6 +4,10 @@ metadata:
   name: {{ include "alertmanager.fullname" . }}
   labels:
     {{- include "alertmanager.labels" . | nindent 4 }}
+{{- if .Values.service.annotations }}
+  annotations:
+{{ toYaml .Values.service.annotations | indent 4 }}
```

```suggestion
    {{- toYaml .Values.service.annotations | nindent 4 }}
```

nit: reads better imho